lang/cs/Org.Apache.REEF.Network/Group/Operators/IBroadcastReceiver.cs

```diff
@@ -26,7 +28,8 @@ public interface IBroadcastReceiver<T> : IGroupCommOperator<T>
         /// <summary>
         /// Receive a message from parent BroadcastSender.
         /// </summary>
+        /// <param name="cancellationSource">The cancellation token for the data reading operation cancellation</param>
         /// <returns>The incoming message</returns>
-        T Receive();
+        T Receive(CancellationTokenSource cancellationSource = null);
```

Comment:
Do we want to enforce cancellation token? Look at .Net BlockingCollection as an example, it has overloading Take() methods, one without cancellationtoken and one with cancellationtoken. Set null would make it optional. If we want to enforce it, I can create a Jira to follow up.